lib/tasks/verify_application_api.rake

```diff
@@ -0,0 +1,110 @@
+class MeasureSerializingApplications
+  attr_accessor :args, :provider, :page, :per_page
+
+  def initialize(args)
+    @args = args
+    @provider = args[:provider_id].present? ? Provider.find(args[:provider_id]) : Provider.first
+    @page = 1
+    @per_page = 50
+  end
+
+  delegate :call, to: :measure_block
+
+  def measure_block
+    lambda {
+      since = Time.zone.iso8601('0001-01-01T01:00:00')
+      presenter = VendorAPI::MultipleApplicationsPresenter.new(
+        '1.1',
+        GetApplicationChoicesForProviders.call(
```

Comment:
Could this just call the wrapping method `get_application_choices_for_provider_since` instead of repeating the includes? I can see this getting out of sync and giving us false security.

Reply:
Done! ♻️